core/src/main/kotlin/no/nav/siftilgangskontroll/core/pdl/PdlAktørIdContext.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.siftilgangskontroll.core.pdl
+
+import kotlinx.coroutines.runBlocking
+import no.nav.security.token.support.core.jwt.JwtToken
+import no.nav.siftilgangskontroll.core.utils.personIdent
+import no.nav.siftilgangskontroll.pdl.generated.hentident.IdentInformasjon
+
+data class PdlAktørIdContext(
+    private val pdlService: PdlService,
+    val borgerToken: String
+) {
+    val identer: List<IdentInformasjon> = runBlocking { pdlService.aktørId(JwtToken(borgerToken).personIdent(), borgerToken) }
+    val pdlPersonContext = PdlPersonContext(pdlService, borgerToken)
+}
+
+fun List<IdentInformasjon>.tilAktørId(): Ident = Ident(first().ident)
+
+data class Ident(internal val value: String)
```

Comment:
Burde denne heller hete AktørId?

Reply:
Kan rename det 👍 
Men den er kalt ident fordi det kan være en folkeregistrert identifikator eller en aktørId.
Men vi bruker jo bare oppslaget til å hente aktørId, så da kan vi kalle den for det :)

Reply:
Aha okei